.circleci/config.yml

```diff
@@ -123,7 +137,8 @@ workflows:
       - test-node-8
       - test-node-10
       - test-node-12
-      - test-node-13 # current
+      - test-node-13
```

Comment:
Shouldn't we get rid of 13 now then?

Reply:
It's not EOL until June. I'm thinking we drop it when we drop 8. We're always waiting for the windows and mac runs on CI anyways, so it doesn't slow us down to keep it